cmd/korg/utils.go

```diff
@@ -121,3 +126,34 @@ func caseAgnosticSort(arr []string) {
 		return strings.ToLower(arr[i]) < strings.ToLower(arr[j])
 	})
 }
+
+func IsOwner(username string) (bool, error) {
+	url := fmt.Sprintf("https://cs.k8s.io/api/v1/search?stats=fosho&repos=*&rng=:20&q=%s&i=fosho&files=OWNERS&excludeFiles=vendor/", username)
+	resp, err := http.Get(url)
+	if err != nil {
+		return false, err
+	}
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return false, err
+	}
+
+	var r client.Response
+
+	err = json.Unmarshal(body, &r)
+	if err != nil {
+		return false, err
+	}
+
+	found := false
+	for _, matches := range r.Results {
+		if len(matches.Matches) != 0 {
+			found = true
```

Comment:
We can add `break` here. 
Or remove the found bool, and `return true, nil` here and `return false, nil` in the end.

Or another way is returning`r.Status.FilesOpened > 0`.